File: BackendEventickNow/src/main/java/com/EventickNow/security/dto/EventoRequest.java
package com.EventickNow.security.dto;

import java.util.Date;

import org.springframework.format.annotation.DateTimeFormat;
import org.springframework.web.multipart.MultipartFile;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class EventoRequest {

	private String nomEvento;
	
	@DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
	private Date fecha;
	
	private String ubicacion;
	
	private String detalles;
	
	private Integer estatus;
	
	private Double costo;
	
	private Integer cantBoletos;
	
	private MultipartFile imagen;
	
	private Integer idOrganizador;
}

File: FrontendEventickNow/src/app/shared/models/eventoRequest.ts
export class EventoRequest {
    nomEvento : any;
    fecha     : any;
    ubicacion : any;
    detalles  : any;
    estatus   : any;
    costo     : any;
    cantBoletos: any;
    nombreImagen:any;
    tipoImagen:any;
    bytesImagen: Uint8Array;
    multipartFile: File;
    imagen: Uint8Array;
    idOrganizador: any
}